<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Объекты газоснабжения (полигон)</string>
  </property>
  <property name="accessibleName">
   <string notr="true">btnSave</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="1">
          <widget class="QLineEdit" name="tbCadastralNumber">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="tbDataSource">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Источник данных</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Состояние объекта</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbObjectType">
           <property name="accessibleName">
            <string notr="true">cbPipeType</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Расположение объекта</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QgsCheckableComboBox" name="cbArrangement">
           <property name="currentText">
            <string/>
           </property>
           <property name="defaultText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="tbDenomination">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Вид объекта</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbObjectState">
           <property name="accessibleName">
            <string notr="true">cbPipeType</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Наименование</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Кадастровый, 
идентификационный номер</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Местоположение</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="tbLocation">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Дополнительные сооружения</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="cbAdditionalStructures"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnAdditionalAttributes">
         <property name="text">
          <string>Доп. атрибуты</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnObjectPassport">
         <property name="text">
          <string>Паспорт объекта</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDownloadFiles">
         <property name="text">
          <string>Файл источник</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnOk">
         <property name="accessibleName">
          <string notr="true">btnSave</string>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="accessibleName">
          <string notr="true">btnCancel</string>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>qgscheckablecombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tbDenomination</tabstop>
  <tabstop>cbObjectState</tabstop>
  <tabstop>cbObjectType</tabstop>
  <tabstop>cbArrangement</tabstop>
  <tabstop>tbLocation</tabstop>
  <tabstop>tbCadastralNumber</tabstop>
  <tabstop>cbAdditionalStructures</tabstop>
  <tabstop>tbDataSource</tabstop>
  <tabstop>btnAdditionalAttributes</tabstop>
  <tabstop>btnObjectPassport</tabstop>
  <tabstop>btnDownloadFiles</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
